masschallenge/django-accelerator
# [AC-9669]: assign legacy users to an HC based on  past confirmed roles

[AC-9669](https://masschallenge.atlassian.net/browse/AC-9669)

How to test Using Django shell
- Fetch all program families of a user via the user program role grant
- pick the `home community` of a program family that has the smallest `assignment_order`
- Ensure it is the same home community set on that user profile.
